components/more-less/more-less.js

```diff
@@ -197,24 +197,29 @@ class MoreLess extends LocalizeCoreElement(LitElement) {
 			return;
 		}
 
+		if (this.expanded) {
+			this.__maxHeight = `${this.__content.scrollHeight}px`;
+			return;
+		}
+
```

Comment:
I think this is unnecessary as the max height should already be set in `__adjustToContent_resize()`

Reply:
Change has been made, thanks!